tests/011-test-docker-root-volume-cleanup.sh

```diff
@@ -21,6 +21,7 @@ test_reset_docker_root_volume() {
 DEVS="$devs"
 VG=$vg_name
 DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=20%FREE
```

Comment:
We dropped DOCKER_ROOT_VOLUME_SIZE as a default, and I thought it better to try multiple defaults.  We probably should have some failed cases.  But we really need cases that tests that this works the new prescribed way.